README.md

```diff
@@ -105,12 +69,43 @@ functions:
         layers:
             - ${bref:layer.php-74}
         events:
+            # Read more at https://www.serverless.com/framework/docs/providers/aws/events/sqs/
             - sqs:
                 arn: arn:aws:sqs:us-east-1:1234567890:my_sqs_queue
                 # Only 1 item at a time to simplify error handling
                 batchSize: 1
 ```
 
+2. Create the handler script (for example `bin/consumer.php`):
+
+```php
+<?php declare(strict_types=1);
+
+use Bref\Symfony\Messenger\Service\Sqs\SqsConsumer;
+
+require dirname(__DIR__) . '/config/bootstrap.php';
+
+$kernel = new \App\Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
+$kernel->boot();
+
+// Return the Bref consumer service
+return $kernel->getContainer()->get(SqsConsumer::class);
+```
+
+3. Register and configure the `SqsConsumer` service:
+
+```yaml
+# config/services.yaml
+services:
+    Bref\Symfony\Messenger\Service\Sqs\SqsConsumer:
+        public: true
```

Comment:
Usually "autowrite: true" is default in this file. But you might want to add it here. 
```suggestion
        public: true
        autowire: true
```

Reply:
Oh yes good point! I'll add it to the other consumers too.